bandit/__init__.py

```diff
@@ -18,3 +18,7 @@
 
 __author__ = metadata.metadata("bandit")["Author"]
 __version__ = metadata.version("bandit")
+# __version__ = metadata.version(__package__)
+# running bandit inside pre-commit we do not get a version here, workaround:
+if __version__ == "0.0.0":
+    __version__ = "latest"
```

Comment:
This seems a bit hacky. I'd rather we get to the root cause on why the semver doesn't get into pre-commit.

Reply:
I agree. This seems like a bug where this is hiding that bug rather than fixing it.

Reply:
My guess is this is a `pbr` problem. If we switched to a more modern build-system we might not have this bug at all.